threadedcomments/views.py

```diff
@@ -2,80 +2,118 @@
 from django.contrib.auth.decorators import login_required
 from django.contrib.contenttypes.models import ContentType
 from django.contrib import messages
-from django.shortcuts import get_object_or_404, render_to_response
```

Comment:
Replaced `render_to_response` with `render`

Everything else in this file is flake8